### ISSUE TITLE
Accessibility: Textarea: Warning and Error information is not conveyed when text area is accessed.  

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3442]
Description: Warning and Error information is not conveyed when text area is accessed.  

Expected: Ideally, such warning and error information should be conveyed when UI is accessed. 

Attached screenshot:

![image](https://user-images.githubusercontent.com/32538291/94519463-26653580-0248-11eb-8ed7-378744802798.png)
